docs/development/testing_configuration_tests.rst

```diff
@@ -27,26 +32,21 @@ For the staging VMs:
 .. code:: sh
 
     make build-debs
-    vagrant up /staging/
+    molecule converge -s libvirt-staging
 
-Running all VMs concurrently may cause performance
-problems if you have less than 8GB of RAM. You can isolate specific
-machines for faster testing:
+Then, to run the tests:
 
 .. code:: sh
-
-    ./testinfra/test.py app-staging
-    ./testinfra/test.py mon-staging
-
-.. note:: The config tests for the ``app-prod`` and ``mon-prod`` hosts are
-          incomplete. Further changes are necessary to run the tests via
-          SSH over Authenticated Tor Hidden Service (ATHS), for both local
-          testing via Vagrant and automated testing via CI.
+   
+   molecule verify -s libvirt-staging
 
 Test failure against any host will generate a report with informative output
-about the specific test that triggered the error. The wrapper script
+about the specific test that triggered the error. Molecule
 will also exit with a non-zero status code.
 
+.. note:: To build and test the VMs with one command, use the Molecule ``test``
+  action: ``molecule test -s libvirt-staging``.
```

Comment:
Using `molecule test` is my usual workflow, as well, however it comes with a cost: if errors are encountered during converge or verify, the VMs will be destroyed. That's quite frustrating as a developer, albeit appropriate for CI. The [docs](https://molecule.readthedocs.io/en/stable-1.22/usage.html?highlight=--destroy) mention a `--destroy` flag:

> The test command supports a `--destroy` argument that will accept the values _always_ (default), _never_, and _passing_. 

We could update the `make staging` target to call `molecule test -s <scenario> --destroy never`, which might be the best compromise. See `devops/create-staging-env` for where the magic happens.

Reply:
Folks may not always be interested in running the infra tests as part of creating VMs though. Updated docs for now.